HIP/hip-1249.md

```diff
@@ -153,6 +153,10 @@ operations not covered by other throttles.
 
 If a precompile/system call exhausts ops mid-transaction, handle as in Execution Outcomes.
 
+## Mirror Node Considerations
+The mirror node computed a value called `gasConsumed` in order to provide a more accurate representation of gas used by
+the evm.  As the `gasUsed` value will accurately reflect the gas used by the evm, the `gasConsumed` value will be deprecated.
```

Comment:
What do we mean by deprecated? Will it be removed? Maybe we should just say its value will match gasUsed and should be ignored going forward? Removing it could break people. Sometimes people interpret deprecated as "don't use this" and sometime as "removed". I think we mean deprecated in the former sense. To remove confusion maybe we say deprecated but not removed for compatibility sake?

Reply:
Yes, I meant not removed but ignored..  The field will not be removed from the mirror node end-point, but should be removed from hash scan.  Currently, `gasConsumed` will not always match `gasUsed` because there is some gas that (like storage costs during deployment) which are missing from  `gasConsumed`.

I'll make an edit to say that going forward, `gasConsumed` will equal `gasUsed`